src/main/webapp/script.js

```diff
@@ -38,6 +38,87 @@ function displayError() {
   alert(alertMessage);
 }
 
+//Array of the (currently 6 for this demo build) 15 listings gathered from the fetch request
+let listingsArray = [];
+
+//Count of the total businesses in the fetch request, used to set a unique id for each card
+let count = 0;
+
+function fetchList() {
+  fetch('/data').then(response => response.json()).then((listings) => {
+    listings.forEach((listing) => {
+      listingsArray.push(createResultCard(listing.name, listing.address, listing.image, listing.rating, listing.website, count));
+      count++;
+    });
+    initialDisplay();
```

Comment:
Does this need to be in the fetch? Can the list be fetched, and the cards be created, then displayed?

Reply:
It was, but we found that the function would actually call the initialDisplay() function before doing the fetch, causing the whole thing to break, so I moved it back inside the fetch.